spec/models/appointment_spec.rb

```diff
@@ -11,6 +11,42 @@
     expect(subject.reference).to eq(subject.booking_request.reference)
   end
 
+  describe '#fulfilment_time_seconds' do
+    subject { build(:appointment) }
+
+    it 'defaults to 0' do
+      expect(described_class.new.fulfilment_time_seconds).to be_zero
```

Comment:
Yeah, in practice that scenario can never happen. If the record has a `proceeded_at` - which incidentally is a user driven thing, not a typical rails timestamp - then it will have entries for these statistics too.

Also to preempt any discussion around this approach - we spoke about pushing a full, serialized version of the `Appointment` and its associated stuff to a queue, and processing these kinds of things out-of-band but that's perhaps something for the future...